my_compassion/static/src/js/my2_prevent_modal_resize_error.js

```diff
@@ -0,0 +1,26 @@
+odoo.define("my_compassion.my2_prevent_modal_resize_error", function (require) {
```

Comment:
Every new JS file, especially one that performs a global override like this, must include a header documentation block for future reference.

Documentation should contain a short summary of the functionality attended by the file, where it is used, and a ticket reference in that case would be needed as well.